publishing/filterAPI/post_dimension_option_test.go

```diff
@@ -48,23 +48,28 @@ func TestSuccessfulPostDimensionOptions(t *testing.T) {
 			os.Exit(1)
 		}
 
-		filterAPI.POST("/filters/{filter_blueprint_id}/dimensions/age/options/28", filterBlueprintID).
+		response := filterAPI.POST("/filters/{filter_blueprint_id}/dimensions/age/options/28", filterBlueprintID).
 			WithHeader(serviceAuthTokenName, serviceAuthToken).
-			Expect().Status(http.StatusCreated)
+			Expect().Status(http.StatusCreated).JSON().Object()
+		validateOptionResponse(*response, filterBlueprintID, "age", "28")
 
 		// Add a duplicate option, this should not be added into the dimension option
-		filterAPI.POST("/filters/{filter_blueprint_id}/dimensions/age/options/28", filterBlueprintID).
```

Comment:
can this test be separated from the below, and the comment changed to something less contradictory, perhaps:
`// Re-adding an existing option should not change the dimension option`